<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 222, 222);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>584</width>
      <height>488</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="busca_nome">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>500</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: white;</string>
     </property>
     <property name="placeholderText">
      <string>Nome ou parte do nome</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="busca_cpf">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>500</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: white;</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>xxx.xxx.xxx-xx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="busca_rg">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>500</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>50</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CPF:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>50</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RG:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>50</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>220</y>
       <width>591</width>
       <height>62</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Limpar todos os campos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(00,0,0);
background-color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buscar">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pesquisar cliente no banco de dados da empresa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(0,0,0);
background-color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_inicial">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Voltar para a tela inicial&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(00,0,0);
background-color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>Tela inicial</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>* Deixar valores em branco para retornar todo o banco de clientes</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consulta de dados</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>busca_rg</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>busca_cpf</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>busca_nome</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
